File: TME1/Couleur/CouleurC/CouleurC/ViewController.m
//
//  ViewController.m
//  CouleurC
//
//  Created by m2sar on 21/09/2016.
//  Copyright © 2016 UPMC. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

NSArray* colorTable;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    colorTable = [ NSArray arrayWithObjects: [UIColor whiteColor], [UIColor blackColor], [UIColor yellowColor], [UIColor redColor], [UIColor greenColor], [UIColor blueColor], nil];
    
    [_stepperOutlet setMinimumValue:0.0];
    [_stepperOutlet setMaximumValue:5.0];
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}




- (IBAction)stepperButton:(id)sender {
    [_ViewWindow setBackgroundColor: colorTable[(int)_stepperOutlet.value]];
}

@end

File: TME1/Couleur/CouleurSwift/CouleurSwift/ViewController.swift
//
//  ViewController.swift
//  CouleurSwift
//
//  Created by m2sar on 26/09/2016.
//  Copyright © 2016 UPMC. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    let colorTable:[UIColor] = [.white, .black, .yellow, .red, .green, .blue]
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        stepperOutlet.minimumValue = 0.0
        stepperOutlet.maximumValue = 5.0
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    
    @IBOutlet weak var colorView: UIView!
    @IBOutlet weak var stepperOutlet: UIStepper!
    
    @IBAction func stepperAction(_ sender: UIStepper) {
        colorView.backgroundColor = colorTable[Int(stepperOutlet.value)]
        
    }
    
    
    
    
    
}

